### DISCUSSION TITLE
Support for Buderus GB192i.2

### DISCUSSION THREAD

**Discussion by t785634** [emsesp/EMS-ESP32#1357] [Q&A]
Is the Buderus GB192i.2 supported by the BBQKees EMS bus gateway? The website says only the GB192i , but the GB192i.2 is a different model.

**Comment by MichaelDvP** [comment]
Yes, it's device-ID 195, shows up as `Condens 5000i/Greenstar 8000/GC9800IW` in release and in dev build as `Condens 5000i/Greenstar 8000/GC9800IW/GB192i.2`.
